srtcore/sync.h

```diff
@@ -211,6 +211,43 @@ std::string FormatTime(const steady_clock::time_point& time);
 /// @returns a string with a formatted time representation
 std::string FormatTimeSys(const steady_clock::time_point& time);
 
+enum eUnit {DUNIT_S, DUNIT_MS, DUNIT_US};
+
```

Comment:
The name `eUnit` is not very self-descriptive.
Maybe `eDurationUnit`?
`DUNIT_MS` can stay as is.